Assets/MRTK/Tools/MSBuild/Scripts/UnityProjectInfo.cs

```diff
@@ -36,6 +36,8 @@ public class UnityProjectInfo
             "Microsoft.MixedReality.Toolkit.Providers.XRSDK.Oculus.Handtracking.Editor",
             "Microsoft.MixedReality.Toolkit.Providers.XRSDK.WindowsMixedReality",
             "Microsoft.MixedReality.Toolkit.Providers.XRSDK",
+            "LeapMotion",
```

Comment:
Since the Leap dependencies aren't 2019-only, we might want to put them into `ExcludedPackageReferences` instead of `PackageReferencesUnity2019`, which behaves similarly but doesn't only run on Unity 2019.